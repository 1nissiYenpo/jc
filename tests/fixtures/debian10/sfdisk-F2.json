[{"disk":"/dev/sdc","free_disk_size":"1 GiB","free_bytes":1073741824,"free_sectors":2097152,"units":"sectors of 1 * 512 = 512 bytes","logical_sector_size":512,"physical_sector_size":512,"partitions":[{"start":4194304,"end":6291455,"sectors":2097152,"size":"1G"}]}]
