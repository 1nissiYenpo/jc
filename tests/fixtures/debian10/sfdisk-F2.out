Unpartitioned space /dev/sdc: 1 GiB, 1073741824 bytes, 2097152 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes

  Start     End Sectors Size
4194304 6291455 2097152   1G
